examples/scripts/compute.py

```diff
@@ -25,18 +26,25 @@
 
 # -------------- USER INPUTS --------------
 # list of scenarios to be evaluated
-scenarios = ["general_add_cost_inputs"]
+scenarios = ["cp_nigeria_usungwe_example"]
```

Comment:
I guess it is was overseen @paulapreuss ?

Reply:
Yes, I'd say this was an oversight leftover from testing.